apps/marketing/src/api/claim-plan/fetcher.ts

```diff
@@ -1,4 +1,5 @@
-import { TClaimPlanRequestSchema, ZClaimPlanResponseSchema } from './types';
+import type { TClaimPlanRequestSchema } from './types';
```

Comment:
is this eslint doing its thing?

Reply:
Nope, it was just me xD
But eslint gave the warnings so I just fixed it